docs/astro.config.mjs

```diff
@@ -15,6 +15,9 @@ export default defineConfig({
     server: {
       force: true,
     },
+    ssr: {
+      noExternal: ['starlight-links-validator'],
+    },
```

Comment:
Why was this change needed? It's only a dev dependency used as a binary script when building the docs to check for broken links, I don't see why Vite would ever resolve an import to this module.